### ISSUE TITLE
README should have at least some description of the different parts of the project and what they do

### ISSUE THREAD

**Issue by NaughtyDog6000** [bgnerdclub/birb#10]
The current plan for the readme is to have a description of what the different structures, components, implementations and synonyms for things are.

this is currently being worked on in the readme branch (linked in development on this issue)

**This will most likely be merged into main wthin the next few days**

**Comment by jw2476** [comment]
I agree, we can write something up on entities, modules, archetypes, etc. as well as a minimal getting started guide.

Is there anything people feel we should add?

**Comment by mavic7** [comment]
In my opinion we should use README to present an informal overview of the project, its’ design values and subsequent architecture whilst leaving the specific intricacies of each module to a dedicated documentation/guide

**Comment by NaughtyDog6000** [comment]
> In my opinion we should use README to present an informal overview of the project, its’ design values and subsequent architecture whilst leaving the specific intricacies of each module to a dedicated documentation/guide 

That's was what I meant, just poorly communicated you can see what I meant in the latest commit to this branch although I did intend to include a list of the core modules and what they do.

**Comment by Alex-Programs** [comment]
It'd be useful to make a systems diagram in https://d2lang.com/ .
